tests/_helpers.py

```diff
@@ -128,3 +128,16 @@ async def execute_command(websocket, command, result_field='result'):
         if 'id' in resp and resp['id'] == command_id:
             assert result_field in resp
             return resp[result_field]
+
+# Wait for a specific response returned from Bidi server
+# event: a dictionary like {'method': 'entryAdded'}
+async def wait_for_event(websocket, event):
```

Comment:
The method name+signature is a bit confusing. Currently there is nothing specific to the event in the method. IMO we should either remove `event` from the method name:
* call it like `wait_for_message` or something, as no event-specific logic is there;
* make the matcher support several levels comparison (like recursive compare);
* reuse the method in the method `execute_command` above, as they do the very same thing

Another option is to make the second argument a flat string `event_name` and check it only in case of an incoming message have a `method` field.

I'm fine with both approaches.

Reply:
updated